testsuite/module-publish/src/packages/simple/sources/Simple.move

```diff
@@ -57,12 +58,56 @@ module 0xABCD::simple {
     // Test simple CPU usage. Loop as defined by the input `count`.
     // Not a true test of CPU usage given the number of instructions
     // used, but a simple reference to computation with no data access.
-    public entry fun loopy(_s: &signer, count: u64) {
+    public entry fun loop_nop(_s: &signer, count: u64) {
```

Comment:
We can have a noop loop which runs out of gas? This would be just a single branch? Or do we detect it somehow in verifier?

Reply:
I am not sure how it works internally, but loop_nop takes different amount of time and gas based on count - so I assume it is not optimized internally